setup.py

```diff
@@ -73,7 +73,7 @@
     packages=[package for package in find_packages() if package.startswith("stable_baselines3")],
     package_data={"stable_baselines3": ["py.typed", "version.txt"]},
     install_requires=[
-        "gym==0.21",  # Fixed version due to breaking changes in 0.22
+        "gym>=0.21",
```

Comment:
I would fix gym version again, unless we are sure no breaking changes is coming afterward...

Reply:
There will be breaking API changes until 1.0, probably in a few more months. Any releases after will still likely result in some breaking changes, e.g. environment version bumps and so on.